.github/workflows/ci.yml

```diff
@@ -23,6 +24,7 @@ env:
 jobs:
   build:
     name: Build and Test
+    if: '!(github.event_name == ''pull_request'' && github.event.pull_request.draft)'
```

Comment:
I think it should be `||` instead of `&&`, no?

|event name == pull_request|draft|and|not|
|----------|-----|---|---|
|true|true|true|false|
|true|false|false|true|
|false|true|false|true|
|false|false|false|true|

Reply:
There's a negation around the whole thing.